docs/settings.rst

```diff
@@ -11,6 +11,8 @@ This extension contributes the following settings:
 
 :needls.pythonPath: Python path used to install `Open-Needs-IDE:needls`. Default system python path `/usr/bin/python` will be used if this setting is not configured.
 
+:needls.extConfPyPath: Extra custom configuration python file like conf.py. This is an absolute path! It's optional, default is None. 
```

Comment:
Must it be an absolute path?
Normally absolute path do not work well in shared env/configs.
Would really like to see this relative as well.